src/auth_service/auth_adapter/core/session_store.py

```diff
@@ -44,10 +44,13 @@ class SessionState(str, Enum):
 class Session(BaseSession):
     """Model for storing user sessions."""
 
-    user_id: str = Field(
+    ext_id: str = Field(
         default=...,
-        description="internal ID of the associated user,"
-        " or external ID if the user is not registered yet",
+        description="External ID of the associated user",
+    )
+    user_id: Optional[str] = Field(
+        default=None,
+        description="internal ID of the associated user, if registered",
```

Comment:
```suggestion
        description="Internal ID of the associated user, if registered",
```

Reply:
Ok, did the same with the other descriptions. Can you approve again?